Frameworks/CoreText/CTFont.mm

```diff
@@ -852,9 +903,8 @@ CFDataRef CTFontCopyTable(CTFontRef font, CTFontTableTag table, CTFontTableOptio
 */
 CFTypeID CTFontGetTypeID() {
     static dispatch_once_t initOnce = 0;
-    dispatch_once(&initOnce,
-                  ^{
-                      __kCTFontTypeID = _CFRuntimeRegisterClass(&__CTFontClass);
-                  });
+    dispatch_once(&initOnce, ^{
+        __kCTFontTypeID = _CFRuntimeRegisterClass(&__CTFontClass);
+    });
     return __kCTFontTypeID;
```

Comment:
?

---
In reply to: [84556024](https://github.com/Microsoft/WinObjC/pull/1145#discussion_r84556024) [](ancestors = 84556024)